Library/ViewModels/PaymentSourceSelected.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+public enum PaymentSourceSelected: Equatable {
+  case paymentSourceId(String)
```

Comment:
Suggestions for better names welcome - as far as I know, this is an ID that comes from GraphQL and represents a particular payment methods stored in our own DB.